scaleway/resource_k8s_pool_beta.go

```diff
@@ -115,6 +115,11 @@ func resourceScalewayK8SPoolBeta() *schema.Resource {
 				Computed:    true,
 				Description: "The Kubernetes version of the pool",
 			},
+			"actual_size": {
```

Comment:
Is `current_size` a better name ?

@remyleone

Reply:
yup current_size sounds better and is more descriptive